.eslintrc.json

```diff
@@ -0,0 +1,58 @@
+{
+  "env": {
+    "node": true
+  },
+  "rules": {
+    "no-var": "error",
+    "prefer-arrow-callback": "warn",
+    "no-unused-vars": [
+      "warn"
+    ],
+    "no-warning-comments": "warn",
+    "no-console": 0,
+    "max-len": [
+      "warn",
+      {
+        "code": 100,
+        "ignoreComments": true,
+        "ignoreUrls": true,
+        "ignoreStrings": true,
+        "ignoreTemplateLiterals": true,
+        "ignoreRegExpLiterals": true
+      }
+    ],
+    "comma-dangle": [
+      "error",
+      "always-multiline"
+    ],
+    "eol-last": [
+      "error",
+      "always"
+    ],
+    "keyword-spacing": [
+      "error"
+    ],
+    "object-curly-spacing": [
+      "error",
+      "always"
+    ],
+    "no-trailing-spaces": "error",
+    "prefer-const": "warn",
+    "operator-linebreak": [
+      "error",
+      "after",
+      {
+        "overrides": {
+          "?": "before",
+          ":": "before",
+          "===": "ignore"
+        }
+      }
+    ],
+    "arrow-parens": [
+      "warn",
+      "as-needed"
+    ],
+    "consistent-return": "error"
+  }
+}
```

Comment:
i'd add a trailing new line (and configure it in your IDE). -> you can see the red arrow pointing at that...

Reply:
do you mean eol-last https://eslint.org/docs/rules/eol-last? It's already there

Reply:
if you notice this file in the code review, you see the marker, that you're missing this in this very file ;)

Reply:
fixed